modules/SofaGeneralDeformable/src/SofaGeneralDeformable/TriangularBendingSprings.inl

```diff
@@ -53,43 +51,38 @@ void TriangularBendingSprings<DataTypes>::TriangularBSEdgeHandler::applyCreateFu
 
         ei.is_activated=false;
         ei.is_initialized=false;
-
-    }
 }
 
 
 
 template< class DataTypes>
-void TriangularBendingSprings<DataTypes>::TriangularBSEdgeHandler::applyTriangleCreation(const sofa::helper::vector<Index> &triangleAdded, const sofa::helper::vector<core::topology::Triangle> &, 
+void TriangularBendingSprings<DataTypes>::applyTriangleCreation(const sofa::helper::vector<Index> &triangleAdded, const sofa::helper::vector<core::topology::Triangle> &, 
     const sofa::helper::vector<sofa::helper::vector<Index> > &, const sofa::helper::vector<sofa::helper::vector<double> > &)
 {
     using namespace core::topology;
-    if (ff)
-    {
-
-        double m_ks=ff->getKs();
-        double m_kd=ff->getKd();
+        double m_ks=getKs();
```

Comment:
```suggestion
        const double ks = getKs();
```
I suggest:
- to remvoe the `m_` suffix as it is not a class member
- to add spaces around `=`
- to make it const

Reply:
same I would like to focus the change on the topology callback only. But I agree a massive cleanup is needed in this class...